src/app/debug/debug-tree/debug-tree.component.ts

```diff
@@ -54,6 +55,7 @@ export class DebugTreeComponent implements OnDestroy {
     highlightOpenFolders: false,
     folderBehaviourOnClick: 'select',
     autoOpenCondition: this.conditionalOpenFunction,
+    determineIconClass: SimpleFileTreeUtil.conditionalCssClass,
```

Comment:
See other comment

Reply:
No the method is passed as a reference here